nix/vast/default.nix

```diff
@@ -108,15 +82,29 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = lib.optional isStatic "pic";
 
-  doCheck = false;
+  doCheck = true;
+  # Concurrent testing is not possible because multiple suites reuse the same
+  # file paths.
+  enableParallelChecking = false;
+  preCheck = ''
+    export LD_LIBRARY_PATH=$PWD/libvast
+    export DYLD_LIBRARY_PATH=$PWD/libvast
+  '';
```

Comment:
Why do we need this?

Reply:
This is for non-static builds. Nix invokes CMake with the option that rpaths get set to the final locations of the install prefix - `BUILD_WITH_INSTALL_RPATH` or something like that - so this is a workaround for `vast-test`.

Reply:
Can you add that as a comment here for the future maintainer?